### ISSUE TITLE
[Problem]: Studentnet0 staat niet tussen de mogelijke netwerken bij de pfsense

### ISSUE THREAD

**Issue by jaelkwakkel** [Windesheim-HBO-ICT/security_workshops#22]
### Workshop

Setup Skylab

### Steps

Opdracht 2 stap 4

### Problem

Wanneer het netwerkinterface gewijzigd wordt in opdracht 2 stap 4 staat studentnet0 hier niet tussen.

**Fix**: Momenteel is de enige bekende oplossing 2 uur wachten. De netwerken worden om de 2 uur uitgedeeld, waarna ze beschikbaar zijn bij het herconfigureren van de netwerkinterface.

**Comment by FemkeSpanner** [comment]
Klopt, ik had vernomen dat het toewijzen van de LAN ook een handmatig proces is. Oplossing hiervoor zou kunnen zijn dat alle studenten voordat ze met skylab kunnen begin opdracht 1 thuis al hebben gedaan.

**Comment by s1177513** [comment]
Het aanmaken van de studentnetten is een automatisch proces, gebeurt 1x in de 2 uur automatisch.
Zie daarvoor ook de Skylab SharePoint pagina.
Het toewijzen van een netwerkadapter aan een machine is een handmatig proces (door de student), echter kan dit ook geautomatiseerd worden d.m.v. Terraform.
